checker/manual/creating-a-checker.tex

```diff
@@ -194,27 +193,26 @@ \subsection{Defining the type qualifiers\label{define-type-qualifiers}}
 
 \begin{Verbatim}
   // Define an annotation for the @NonNull type qualifier.
-  @TypeQualifier
   @Target({ElementType.TYPE_USE, ElementType.TYPE_PARAMETER})
   public @interface NonNull { }
 \end{Verbatim}
 
 \noindent
-Write the \refqualclass{framework/qual}{TypeQualifier} meta-annotation on the annotation definition
-to indicate that the annotation represents a type qualifier
-and should be processed by the checker.
-Also write a \sunjavadocanno{java/lang/annotation/Target.html}{Target}
-meta-annotation to indicate where the annotation may be written.
+
+Write a \sunjavadocanno{java/lang/annotation/Target.html}{Target}
+meta-annotation on the annotation definition to indicate where the annotation may be written.
+All type qualifiers that users can use should have the value \<ElementType.TYPE\_USE> and/or \<ElementType.TYPE\_PARAMETER>.
+The checker-framework automatically considers any annotation that has one of these values as a type qualifier.
```

Comment:
Remove "as" which is not grammatical (unless you change "considers" to "treats").